admin-dev/themes/default/template/helpers/form/form.tpl

```diff
@@ -342,7 +342,7 @@
 										{$input.desc = null}
 									{else}
 										<select name="{$input.name|escape:'html':'utf-8'}"
-												class="{if isset($input.class)}{$input.class|escape:'html':'utf-8'}{/if} fixed-width-xl"
+												class="{if isset($input.class)}{$input.class|escape:'html':'utf-8'}{/if}"
```

Comment:
Hello this seems to make sense but I wonder why the `fixed-width-xl` was added in the past. Maybe to prevent wrongly implemented module to destroy the webpage rendering?

Wdyt @Hlavtox ?

Reply:
I don't know, but I feel @Amit-Kumar-Tiwari-Webkul's change is correct. What does @NeOMakinG think about this?

Reply:
https://github.com/NeOMakinG/PrestaShop/commit/94a219b376626d19ebf2374c64388055f3df4daa

Looks like it was a design change introduced 8 years ago, I would go for @Amit-Kumar-Tiwari-Webkul proposal!